connectors/src/connectors/notion/temporal/workflows.ts

```diff
@@ -246,18 +255,22 @@ export async function notionSyncWorkflow(
   );
 }
 
+type SyncWorkflowResult = {
```

Comment:
This seems like an unecessary type? Why not just UpsertActivityResult[] direclty?